src/applications/third-party-apps/components/SearchResult/index.jsx

```diff
@@ -0,0 +1,147 @@
+            {/* App Name */}
+            <h3
+              className="vads-u-margin--0 vads-u-margin-bottom--0p5"
+              data-e2e-id="result-title"
+              id={item?.id}
+            >
+              {item?.name}
+            </h3>
+
+            {/* Category and Platform */}
+            <p className="vads-u-margin--0">
+              {join(item?.categories, ', ') || 'Unknown category'} app available
+              for {join(item?.platforms, ', ') || 'unknown platforms'}
+            </p>
+          </div>
+
+          {/* App URL */}
+          <a
+            className="usa-button usa-button-secondary vads-u-width--auto"
+            href={item?.appURL}
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            Find app
+          </a>
+        </div>
+
+        {/* Toggle More Info */}
+        <div className="learn-more">
+          <button
+            className="va-button-link vads-u-text-decoration--none vads-u-border-color--link-default vads-u-border-style--dotted vads-u-border-bottom--1px vads-u-margin-top--1p5"
+            onClick={() => setShow(!show)}
+            type="button"
+          >
+            Learn about {item?.name} <i className="fa fa-cenvron-down" />
+          </button>
+        </div>
+
+        {show && (
+          <>
+            <hr />
+
+            {/* Description */}
+            {item?.description && (
+              <>
+                <h4>About this app:</h4>
+                <p className="vads-u-margin-bottom--0">{item?.description}</p>
+              </>
+            )}
+
+            {/* Permissions */}
+            {!isEmpty(item?.permissions) && (
+              <>
+                <h4>{item?.name} asks for:</h4>
+                <ol className="vads-u-margin--0 vads-u-padding-left--2p5">
+                  {map(item?.permissions, permission => (
+                    <li key={permission}>{permission}</li>
+                  ))}
+                </ol>
+              </>
+            )}
+
+            {/* Legal Links */}
+            <h4>More information:</h4>
+            <a
+              href={item?.privacyPolicyURL}
+              rel="noopener noreferrer"
+              target="_blank"
+            >
+              View privacy policy
+            </a>
+            <a
+              className="vads-u-margin-top--1"
+              href={item?.termsOfServiceURL}
+              rel="noopener noreferrer"
+              target="_blank"
+            >
+              View terms of service
+            </a>
+            <a
+              className="vads-u-margin-top--1"
+              href={encodeURIComponent(
+                `mailto:api@va.gov?subject=Report ${item?.name} to VA`,
+              )}
+              rel="noopener noreferrer"
+            >
+              Report this app to VA
+            </a>
+          </>
+        )}
+      </li>
+    );
+  }
+}
+
+export default SearchResult;
```

Comment:
I think longterm the plan if for Auth Exp to build this component (bc I think they will ship their product first) then we should just leverage what they built. I _think_, too much up in the air to actually say. But just a side note that we may want to let them know and vice versa